tests/targets/test_android.py

```diff
@@ -93,11 +94,8 @@ def init_target(self, options=None):
         spec = []
         for line in default_spec:
             if line.strip():
-                key = line.split()[0]
-
-                if key.startswith('#'):
-                    key = key[1:]
-
+                match = re.search(r'[#\s]?([a-z_\.]+)', line)
+                key = match and match.group(1)
```

Comment:
More accurate (handle spaces just after the `#` comment), yet more concise version.
Works on matching:
```
#config.option = foo
```
As well as:
```
# config.option = foo
```
Both would make `key = "config.option"`